robot/robot.py

```diff
@@ -18,13 +21,13 @@ class Robot:
 
     ROBOTD_ADDRESS = "/var/robotd"
 
-    def __init__(self, robotd_path=ROBOTD_ADDRESS):
+    def __init__(self, robotd_path:str=ROBOTD_ADDRESS) -> None:
```

Comment:
I think this wants a space before the `str` and after the colon: `robotd_path: str=...`.

Reply:
As does the linter!